cmake/modules/OpenCL.cmake

```diff
@@ -13,6 +13,10 @@ if(USE_OPENCL)
   file(GLOB RUNTIME_OPENCL_SRCS src/runtime/opencl/*.cc)
   list(APPEND TVM_RUNTIME_LINKER_LIBS ${OpenCL_LIBRARIES})
   list(APPEND RUNTIME_SRCS ${RUNTIME_OPENCL_SRCS})
+  if($ENV{XILINX_SDX})
```

Comment:
Would this conflict with runtime SDK?

Reply:
When we have another SDK, we can still compile tvm and find multiple OpenCL platforms with clGetPlatformInfo().  Currently, tvm cannot handle this case but it's another problem:
  https://github.com/dmlc/tvm/blob/1e66d3c/src/runtime/opencl/opencl_device_api.cc#L242
I think of sending another PR to support it.

Anyway, I'll remove this if condition since cmake can compile tvm for sdaccel in either way.  It is not necessary to use headers and libraries in the Xilinx toolchain at compile time.